Automattic/wp-calypso
# SEO Tools: add archive_title page title token

#### Changes proposed in this Pull Request

Related (prerequisite): https://github.com/Automattic/jetpack/pull/20920

This PR:

* Supports adding & previewing a more generic `archive_title` token for custom Archive page titles.
* Hides the `Date` token button for Archives (but we still tokenize existing [date] usage).
* The `Date` button will only be replaced with the `Archive Title` button on Jetpack versions `>= 10.2-alpha`

#### Testing instructions

* See: https://github.com/Automattic/jetpack/pull/20920

---

Screenshot for Translators:

![Markup on 2021-09-10 at 13:33:55](https://user-images.githubusercontent.com/15803018/132907722-5dfe80ff-f14f-4c7b-abba-8f20301e3a39.png)
